recipes/assimp/3.3.1/conanfile.py

```diff
@@ -0,0 +1,128 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class Assimp(ConanFile):
+    name = "assimp"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/assimp/assimp"
+    description = "A library to import and export various 3d-model-formats including scene-post-processing to generate missing render data."
+    topics = ("conan", "assimp", "3d")
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```
https://spdx.org/licenses/BSD-3-Clause.html